src/styles/styles.scss

```diff
@@ -18,9 +18,81 @@
 @import "components/search_bar";
 @import "components/star";
 
+* {
```

Comment:
please drop all changes that are not relevant here, this PR is about making front end build in Docker. This is very hard to review with so many unrelated changes present too 🙏